common/changes/@itwin/ecschema-rpcinterface-tests/Frank-Fullstack-tests-client_2024-12-09-19-57.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ecschema-rpcinterface-tests",
+      "comment": "update package.json",
```

Comment:
leave the comment blank, this PR is changing the tests and is not user-facing, so no need for comments

Reply:
> leave the comment blank, this PR is changing the tests and is not user-facing, so no need for comments

fixed